### PR TITLE
6.2 sdk

### PR DESCRIPTION
**Pull request by Christina-Kang**
Remove customSDK
Updated versions

Verifed:

- [ ] Build and test CI passes
- [ ] History and readme updated to reflect changes
- [ ] Package version updated according to semantic versioning rules
- [ ] Tests modified or added, when applicable
- [ ] Updated code owners file, when applicable
- [ ] Read the [PR checklist](https://github.com/Azure/service-fabric-cli/wiki/Checklist)

### CODE REVIEW

File: src/sfctl/custom_app_type.py [hunk 1]
```diff
@@ -68,7 +68,7 @@ def provision_application_type(client, #pylint: disable=too-many-locals,invalid-
             no_wait,
             application_type_build_path=application_type_build_path)
 
-    api_version = "6.1"
+    api_version = "6.2"
```
Comment (samedder) [inline src/sfctl/custom_app_type.py]: Why are these changes in the remove SDK PR? Can you make these in a separate PR?

**Comment by samedder** [comment]
You need to also undo the changes that are breaking the CI. If the CI is broken constantly then there's no value add. The point of CI is its designed to prevent changes that will break the build in master.

**Comment by Christina-Kang** [comment]
@samedder this PR would break CI because SDK wasn't published when the PR was created.

**Comment by Christina-Kang** [comment]
Closing and splitting into separate PRs
